playbooks/roles/hammer_devel/tasks/hammer_config.yml

```diff
@@ -19,6 +19,20 @@
         :username: {{ hammer_devel_username }}
         :password: {{ hammer_devel_password }}
 
+- name: 'Configure hammer-cli-foreman-admin'
+  blockinfile:
+    dest: ~/.hammer/cli.modules.d/foreman_admin.yml
+    create: true
+    block: |
+      :foreman_admin:
+        :enable_module: true
```

Comment:
A thought, we could store this information with the definition of the plugin and then make this generic, e.g.:

```
- name : "Configure {{ item.name }}"
  blockinfile:
    dest: "~/.hammer/cli.modules.d/{{ item.name }}.yml"
    create: true
    block: {{ item.config }}
  with_dict: "{{ hammer_plugins }}"
```

```
hammer_plugins:
  - name: foreman_admin
    repo: theforeman/foreman_admin
    config: |
      :foreman_admin:
         :enable_module: true
```